TAO/TAO_IDL/be/be_visitor_interface/interface_sh.cpp

```diff
@@ -115,7 +115,7 @@ be_visitor_interface_sh::visit_interface (be_interface *node)
 
   // Copy constructor and destructor.
   *os << class_name.c_str () << " (const "
-      << class_name.c_str () << "& rhs);" << be_nl
+      << class_name.c_str () << "& rhs) = default;" << be_nl
```

Comment:
`rhs` should be removed, will trigger new warnings with some compilers

Reply:
Even for a declaration with no function body?

Reply:
Yes, something I did encounter when working on TAOX11, not sure which compilers, very likely one of the older gcc compilers. You could leave it and the scoreboard will tell whether we still support one of those broken ones, if so, it can be removed easily at that moment.